### ISSUE TITLE
'Move widget' option directly from edit favorites screen

### ISSUE THREAD

**Issue by meher2505** [8bitPit/Niagara-Issues#1443]
Hi Peter,

I believe there was an option in the past where you could move the widget when you're editing the favorites (selecting the favorite apps to be placed on the home screen).

There is no such option anymore, and now everytime we are forced to go to Niagara settings -> Features -> Niagara widget -> Move widget.

Sounds like a long process compared to earlier. Was it intentionally removed from the favorites screen or is it a bug ?

**Comment by Just-SKY** [comment]
It isn't the necessity to do that because you don't move the widget everyday or every moment. When people placed the widget in their favorite position, they don't tend to move it from there. So, it's not something that relevant in my opinion.

**Comment by 8bitPit** [comment]
The command was intentionally removed but we might show a dialog with the same Niagara widget settings when you long-press on Niagara's inbuilt widget in the future.

**Comment by meher2505** [comment]
Hello Peter,

Thank you for responding to my request.

I am a huge fan of all the work you do. I love Niagara Launcher and have
installed it when it was just budding and I looked everywhere to support
your work. When I saw the Pro option, I immediately bought it right away.
Proud to see that it has a huge community now. I myself have recommended it
to a lot of my friends. I genuinely appreciate your hardwork and wish you
all the very best with you upcoming projects.

Regarding my request, I understand now. Yes it would be helpful, IMO, to
have it as an option upon long press.

Thank you and have a good day !

Meher
